src/univalue/.travis.yml

```diff
@@ -26,6 +25,7 @@ addons:
       - pkg-config
 
 before_script:
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew uninstall libtool; brew install libtool; fi
```

Comment:
@paveljanik The issue with homebrew was considered "mostly fixed" (https://github.com/Homebrew/legacy-homebrew/issues/43874#issuecomment-238809893) by some of the workarounds. If the maintainers of that project don't consider it that imporatnt, I think it is fine to just apply a trivial and low risk workaround to fix a travis issue.

Also note that the content of this .travis.yml is of no relevance in the bitcoin repo.